### PR TITLE
changed file structure and DB query

### PR DESCRIPTION
**Pull request by namanahuja**

### CODE REVIEW

**Comment by namanahuja** [comment]
Updated the main layout

**Comment by namanahuja** [comment]
added functionality to change templates

**Comment by namanahuja** [comment]
added functionality to add templates

todo: check if the set already exists to avoid error

File: src/controllers/update.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ });
+
+ module.exports = function(req, res) {
+
+     if (req.params.id != '') {
+
+
+
+
+         con.query('UPDATE notification_type_notification_medium_mapping SET send_template=? WHERE id=?', [req.body.updatetext, req.params.id], function(err, rows) {
```
Comment (sahil-proptiger) [inline src/controllers/update.js]: all queries should be present in a service

File: src/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+
+ module.exports = function(req, res) {
+
+
+
+    
+
+     con.query('SELECT t1.id AS id, t1.notification_type_id AS notification_type_id, t2.name AS mediumname, t3.name AS notificationname, t1.send_template AS template FROM notification_type_notification_medium_mapping t1, notification_medium t2, notification_type t3 WHERE t2.id=t1.notification_medium_id AND t3.id=t1.notification_type_id ORDER BY t2.name DESC', function(err, rows) {
```
Comment (sahil-proptiger) [inline src/controllers/index.js]: all queries should be present in a service

**Comment by namanahuja** [comment]
Made some changes in services

File: app.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+var bodyParser = require('body-parser');
+var urlencodedParser = bodyParser.urlencoded({
+    extended: false
+});
+
+
+var app = express();
+
+app.get('/', require('./src/services/mainService'))
```
Comment (sahil-proptiger) [inline app.js]: use homeController

File: app.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+app.use(express.static('public'));
+
+app.post('/update/:id', urlencodedParser, require('./src/services/updateService') );
```
Comment (sahil-proptiger) [inline app.js]: app.use(urlencodedParser)

File: app.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+app.get('/add/:id', require('./src/controllers/add'));
```
Comment (sahil-proptiger) [inline app.js]: remove get

**Pull request closed without merging**